ProjectK.Business/ExperiencePoints/ExperienceService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Logging;
+using ProjectK.Business.Characters;
+using ProjectK.Contracts;
+using System;
+using System.Threading.Tasks;
+
+namespace ProjectK.Business.ExperiencePoints
+{
+    class ExperienceService : IExperienceService
+    {
+        private readonly IExperienceRepository _repository;
+        private readonly ICharacterRepository _characterRepository;
+        private readonly ILogger _logger;
+
+        public ExperienceService(IExperienceRepository repository, ICharacterRepository characterRepository, ILogger logger)
+        {
+            _repository = repository;
+            _characterRepository = characterRepository;
+            _logger = logger;
+        }
+
+
+        public async Task<ExperienceDto> GetCharacterLevel(int charId)
+        {
+            _logger.LogInformation("Get character level: {charId}", charId);
+
+            var charLevel = await _repository.GetCharacterLevel(charId);
+
+            return charLevel?.ToDto();
+        }
+
+        public Task<ExperienceDto> GetLevelUp(int charId)
+        {
+            _logger.LogInformation("Is character ready to levelUp: {charId}", charId);
+            var currentXp = _repository.GetCharacterCurrentXp(charId);
+
+        }
+
+        public Task<ExperienceDto> GetXpToLevel(int charId)
+        {
+            throw new System.NotImplementedException();
+        }
+        public Task<ExperienceDto> GenerateExperienceToLevel(ExperienceDto model)
```

Comment:
Not sure what this method even does.

Reply:
It's supposed to be a method to generate the ExperienceToLevel table in DB if there isn't one

Reply:
But I don't know if it should be here or not